.github/workflows/release.yaml

```diff
@@ -0,0 +1,45 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Unshallow
+        run: git fetch --prune --unshallow
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13.x
```

Comment:
This version should match the version in the `go.mod`.

Reply:
Got it. Thanks.